### ISSUE TITLE
AttributeError: module 'google' has no attribute 'Plugin'

### ISSUE THREAD

**Issue by Giando** [maldevel/EmailHarvester#26]
Congratulations for the project @maldevel!
I only found a small mistake:
```
Traceback (most recent call last):
  File "./EmailHarvester.py", line 262, in <module>
    app = EmailHarvester(userAgent, args.proxy)
  File "./EmailHarvester.py", line 105, in __init__
    plugins[fname] = mod.Plugin(self, {'useragent':userAgent, 'proxy':proxy})
AttributeError: module 'google' has no attribute 'Plugin'
```
Many thanks!

**Comment by eth0izzle** [comment]
@maldevel yout fix #27 doesn't actually fix the issue, it causes the issue.

It's because of this line:

   mod = __import__(fname)

`fname` = google, which will try and import the library first, rather than EmailHarvester's google.py. You want to do:

    mod = __import__(fname, fromlist=[''])

Which will ensure it loads from local libs first.

**Comment by maldevel** [comment]
@eth0izzle hi mate. could you please send a pull request? I am really out of free time these days.
